image_width: 640
image_height: 480
camera_name: narrow_stereo/right
camera_matrix:
  rows: 3
  cols: 3
  data: [493.202830, 0.000000, 314.228079, 0.000000, 493.320355, 242.674364, 0.000000, 0.000000, 1.000000]
distortion_model: plumb_bob
distortion_coefficients:
  rows: 1
  cols: 5
  data: [0.147239, -0.274312, 0.007788, 0.003329, 0.000000]
rectification_matrix:
  rows: 3
  cols: 3
  data: [0.999165, -0.004654, 0.040582, 0.005116, 0.999923, -0.011277, -0.040526, 0.011476, 0.999113]
projection_matrix:
  rows: 3
  cols: 4
  data: [537.050729, 0.000000, 301.996452, 94.196339, 0.000000, 537.050729, 254.997971, 0.000000, 0.000000, 0.000000, 1.000000, 0.000000]
